### PR TITLE
Output test summary for storage perf test

### PR DESCRIPTION
**Pull request by LiliDeng**
Test result
```
Test Run On           : 03/14/2019 04:48:24
ARM Image Under Test  : canonical : ubuntuserver : 18.04-lts : Latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:1:8

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 PERF-STORAGE-4K-IO                                                                PASS                54.75 
       Mode=randread : block_size=4K q_depth=1 iops=431.003793 
       Mode=randread : block_size=4K q_depth=2 iops=872.728364 
       Mode=randread : block_size=4K q_depth=4 iops=1759.429072 
       Mode=randread : block_size=4K q_depth=8 iops=3569.309538 
       Mode=randread : block_size=4K q_depth=16 iops=5194.683352 
       Mode=randread : block_size=4K q_depth=32 iops=5290.879676 
       Mode=randread : block_size=4K q_depth=64 iops=5781.336468 
       Mode=randread : block_size=4K q_depth=128 iops=5923.131249 
       Mode=randread : block_size=4K q_depth=256 iops=5848.122122 
       Mode=randwrite : block_size=4K q_depth=1 iops=300.551330 
       Mode=randwrite : block_size=4K q_depth=2 iops=620.896995 
       Mode=randwrite : block_size=4K q_depth=4 iops=1282.245310 
       Mode=randwrite : block_size=4K q_depth=8 iops=2647.149726 
       Mode=randwrite : block_size=4K q_depth=16 iops=4815.232273 
       Mode=randwrite : block_size=4K q_depth=32 iops=5594.488395 
       Mode=randwrite : block_size=4K q_depth=64 iops=5783.435503 
       Mode=randwrite : block_size=4K q_depth=128 iops=5380.343826 
       Mode=randwrite : block_size=4K q_depth=256 iops=5700.017266 
```

### CODE REVIEW

File: Testscripts/Windows/PERF-STORAGE-MULTIDISK-RAID0-FIO.ps1 [hunk 1]
```diff
@@ -268,7 +268,19 @@ function Main {
             -OutPerfResultFile $outPerfResultFile
 
         $fioPerfResults = Get-FioPerformanceResults $outPerfResultFile
-
+        #Get current test modes from constants file
+        $modes = Run-LinuxCmd -ip $allVMData.PublicIP -port $allVMData.SSHPort `
```
Comment (ader1990) [inline Testscripts/Windows/PERF-STORAGE-MULTIDISK-RAID0-FIO.ps1]: you can get the modes straight from the fioPerfResults, you don t need to do another ssh

**Comment by LiliDeng** [comment]
Running testing against updated code.

**Review by ader1990 — APPROVED** [review]
looks good to me

**Comment by LiliDeng** [comment]
Thanks! @ader1990

**Comment by LiliDeng** [comment]
Test done against updated code
```
[LISAv2 Test Results Summary]
Test Run On           : 03/14/2019 09:24:53
ARM Image Under Test  : canonical : ubuntuserver : 18.04-lts : Latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:59

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 PERF-STORAGE-4K-IO                                                                PASS                51.55 
	Mode=randread : block_size=4K q_depth=1 iops=484.483437 
	Mode=randread : block_size=4K q_depth=2 iops=978.820141 
	Mode=randread : block_size=4K q_depth=4 iops=1951.687310 
	Mode=randread : block_size=4K q_depth=8 iops=3809.222544 
	Mode=randread : block_size=4K q_depth=16 iops=5043.592752 
	Mode=randread : block_size=4K q_depth=32 iops=5596.500700 
	Mode=randread : block_size=4K q_depth=64 iops=5781.107950 
	Mode=randread : block_size=4K q_depth=128 iops=5922.076712 
	Mode=randread : block_size=4K q_depth=256 iops=6004.892489 
	Mode=randwrite : block_size=4K q_depth=1 iops=366.530890 
	Mode=randwrite : block_size=4K q_depth=2 iops=744.858436 
	Mode=randwrite : block_size=4K q_depth=4 iops=1501.133318 
	Mode=randwrite : block_size=4K q_depth=8 iops=3133.427774 
	Mode=randwrite : block_size=4K q_depth=16 iops=4827.942301 
	Mode=randwrite : block_size=4K q_depth=32 iops=5583.760190 
	Mode=randwrite : block_size=4K q_depth=64 iops=5782.139868 
	Mode=randwrite : block_size=4K q_depth=128 iops=5751.118774 
	Mode=randwrite : block_size=4K q_depth=256 iops=5650.760834 
```

**Pull request merged**